addon/components/one-way-select.js

```diff
@@ -165,7 +165,8 @@ const OneWaySelectComponent = Component.extend(DynamicAttributeBindings, {
       foundOption = options.find(findOption);
     }
 
-    if (optionTargetPath) {
+    if (optionTargetPath &&
+      foundOption !== undefined) {
```

Comment:
With `isPresent()`, won't `'  '` be `false` which from a users point of view could be validly used inside an options object?

http://emberjs.com/api/#method_isPresent